drivers/can/Kconfig.mchp_mss

```diff
@@ -0,0 +1,9 @@
+# Copyright (C) 2025 embedded brains GmbH & Co. KG
+# SPDX-License-Identifier: Apache-2.0
+
+config CAN_MCHP_MSS
+	bool "Microchip PolarFire SoC CAN driver"
+	depends on DT_HAS_MICROCHIP_MPFS_CAN_ENABLED
```

Comment:
small nit: can you add `default y` under `bool "Microchip PolarFire SoC CAN driver"` so that a user only needs to set `CONFIG_CAN=y` in their project please